packages/jellyfish-network/src/Network.ts

```diff
@@ -27,6 +27,12 @@ export interface Network {
   scriptHashPrefix: 0x5a | 0x80 | 0xc4
   /** For message signing. */
   messagePrefix: '\x15Defi Signed Message:\n'
+
+  /** [HYDRA] */
+  ports: {
+    rpc: number
+    p2p: number
+  }
```

Comment:
Hmm, let's move this to another PR? Since it can be standalone.

Reply:
PR for this is here: https://github.com/JellyfishSDK/jellyfish/pull/1907